roles/pulp-devel/templates/alias.bashrc.j2

```diff
@@ -254,3 +254,24 @@ pminio(){
     _paction restart
 }
 _pminio_help="Switch to minio for S3 testing. For stopping it: pminio stop"
+
+buildgalaxyui(){
+    CURRENT_DIR=$(pwd)
+
+    if [ -d $HOME/devel/ansible-hub-ui ]
+    then
+        cd $HOME/devel/ansible-hub-ui
+    else
+        cd $HOME/devel
+        git clone https://github.com/ansible/ansible-hub-ui.git
+        cd ansible-hub-ui
+    fi
+
+    workon pulp
+    STATIC_DIR=$(dynaconf list -k STATIC_ROOT | grep STATIC_ROOT | awk '{ print $2 }' | tr -d "'")
+    npm install
+    npm run build-standalone
+    cp -TR dist/ $STATIC_DIR/galaxy_ui
```

Comment:
Just noticed that copying straight after the build doesn't work, 
I have to run `npm run start` first, as it seems to compile:
```
(pulp) [vagrant@pulp3-source-fedora30 ansible-hub-ui]$ npm run start

> ansible-hub-ui@0.1.0 start /home/vagrant/devel/ansible-hub-ui
> NODE_ENV=development webpack-dev-server --config custom.dev.config.js

~~~Using variables~~~
Root folder: undefined
Current branch: master
Beta branches: master,qa-beta,ci-beta,prod-beta
Using deployments: apps
Public path: /apps/automation-hub/
App entry: /home/vagrant/devel/ansible-hub-ui/src/entry-dev.js
~~~~~~~~~~~~~~~~~~~~~
Overriding app entry for standalone mode.
New entry: /home/vagrant/devel/ansible-hub-ui/src/entry-standalone.tsx
ℹ ｢wds｣: Generating SSL Certificate
ℹ ｢wds｣: Project is running at https://localhost:8002/
ℹ ｢wds｣: webpack output is served from /apps/automation-hub/
ℹ ｢wds｣: Content not from webpack is served from /home/vagrant/devel/ansible-hub-ui/dist
ℹ ｢wds｣: 404s will fallback to /index.html
ℹ ｢atl｣: Using typescript@3.7.2 from typescript
ℹ ｢atl｣: Using tsconfig.json from /home/vagrant/devel/ansible-hub-ui/tsconfig.json
ℹ ｢atl｣: Checking started in a separate process...
ℹ ｢atl｣: Time: 4888ms
ℹ ｢wdm｣: Hash: 50773022e1ce10aa789f
Version: webpack 4.41.2
Time: 50123ms
Built at: 03/30/2020 9:59:43 PM
ℹ ｢wdm｣: Compiled successfully.
```

Reply:
If I copy it before I run `npm run start`
it breaks because I declared `/apps/automation-hub/` on nginx and it tries to get `App.js` and `vendor.js` from `/beta/apps/automation-hub/`

Reply:
That calls `webpack-dev-server` though. Won't that leave a webserver process running?

https://github.com/ansible/ansible-hub-ui/blob/master/package.json#L100

Reply:
it does, that's why I don't know how to solve it. @newswangerd said it could be a bug, and he will look into it

Reply:
fixed here: https://github.com/ansible/ansible-hub-ui/pull/114